### PR TITLE
Support wasm, delay for v1.1.x

### PR DESCRIPTION
**Pull request by RogerKSI**
## Description
- support ignore ack
- support wasm
- support delay

Note: **cherry-pick from master**

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules).
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: config_example.toml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# The global section has parameters that apply globally to the relayer operation.
+[global]
+
+# Specify the verbosity for the relayer logging output. Default: 'info'
+# Valid options are 'error', 'warn', 'info', 'debug', 'trace'.
+log_level = 'trace'
+
+
```
Comment (taobun) [inline config_example.toml:8]: Should add our new config (delay, and ignore ack)?

File: crates/relayer/src/event/rpc.rs [hunk 2]
```diff
@@ -116,6 +116,7 @@ use super::{ibc_event_try_from_abci_event, IbcEventWithHeight};
 pub fn get_all_events(
     chain_id: &ChainId,
     result: RpcEvent,
+    ignore_port_channel: &Vec<PortChannelId>,
```
Comment (taobun) [inline crates/relayer/src/event/rpc.rs]: Fix mai?

**Review by taobun — CHANGES_REQUESTED** [review]

File: config_example.toml (hunk 1 above)
Comment (colmazia) [reply config_example.toml:8]: There are ignore ack here, but yeah, should add delay here.

File: crates/relayer/src/event/rpc.rs (hunk 2 above)
Comment (colmazia) [reply crates/relayer/src/event/rpc.rs]: Nan si

**Pull request merged**